data/json/npcs/TALK_MARLOSS_VOICE.json

```
@@ -0,0 +1,83 @@
+[
+  {
+    "type": "effect_type",
+    "id": "u_offered_marloss_meal",
+    "//": "This flag limits feast offers."
+  },
+  {
+    "type": "effect_type",
+    "id": "u_given_berry",
+    "//": "This flag limits given berries to one per 12h."
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_MARLOSS_VOICE",
+    "dynamic_line": [
+      "Yes, friend?",
+      "Your travels be fruitful, friend.",
+      "May you find your peace, traveler.",
+      "We might have lost everything, but hope remains.",
+      "The earth flourish beneath our paths",
+      "Unity of spirit, of mind, and body...",
+      "Look for the bonds which define you, and act in accord."
+    ],
+    "responses": [
+      {
+        "text": "This place is dangerous, what are you doing here?",
+        "topic": "TALK_MARLOSS_VOICE_BANKET",
+        "condition": { "not": { "u_has_effect": "u_offered_marloss_meal" } },
+        "effect": { "u_add_effect": "u_offered_marloss_meal", "duration": "25920" }
+      },
+      { "text": "Who are you?", "topic": "TALK_MARLOSS_VOICE_A_PRIEST" },
+      { "text": "I gotta go.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_MARLOSS_VOICE_BANKET",
+    "dynamic_line": "Dangerous?  It may look different, but this land cares and provides for us.  We are celebrating with a feast, in fact.  Do you care to join us? ",
+    "responses": [
+      { "text": "Well sure", "topic": "TALK_MARLOSS_VOICE_OFFER_BERRY" },
+      { "text": "I... rather not.", "topic": "TALK_MARLOSS_VOICE" },
+      { "text": "I'm sorry... I have places to be.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_MARLOSS_VOICE_A_PRIEST",
+    "dynamic_line": "I'm a priest or guide of a sort.  I sing the hymns along my companions so that we may learn to live in unity, both with each other and with our ailing world.",
```

```suggestion
    "dynamic_line": "I'm a priest or guide of a sort. I sing the hymns along my companions so that we may learn to live in unity, both with each other and with our ailing world.",
```
2 spaces after full stop.